test/test_main.py

```diff
@@ -138,6 +139,64 @@ def __init__(self, framework):
         self.assertFalse(warn_cm)
 
 
+@patch('sys.argv', new=("hooks/config-changed",))
```

Comment:
It feels weird to me to patch a list with a tuple, but I realize its probably just fine

Reply:
because `new` is only instantiated once i didn't want to have a mutable thing there (could've used `new_callable` instead)